receiver/splunkhecreceiver/config.go

```diff
@@ -9,13 +9,22 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/splunk"
 )
 
+type SplittingStrategy string
+
+const (
+	Line SplittingStrategy = "line"
+	None SplittingStrategy = "none"
```

Comment:
We follow this naming pattern within the collector's codebase. Otherwise, `Line` and `None` consts are pretty ambiguous 

```suggestion
	SplittingStrategyLine SplittingStrategy = "line"
	SplittingStrategyNone SplittingStrategy = "none"
```